oak_functions/sdk/oak_functions/tests/integration_test.rs

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use lazy_static;
+use maplit::hashmap;
+use oak_functions_abi::proto::{Request, Response};
+use oak_functions_loader::{logger::Logger, lookup::LookupData, server::WasmHandler};
+use std::sync::Arc;
+use tokio;
+
+lazy_static::lazy_static! {
+    static ref RUNTIME: tokio::runtime::Runtime = tokio::runtime::Runtime::new().unwrap();
+}
+
+#[test]
+fn test_sdk() {
+    let manifest_path = "/workspace/oak_functions/sdk/oak_functions/tests/module/Cargo.toml";
```

Comment:
This hard-coded path only works inside the docker image. It is better to programatically get the path to the working directory as we do in other tests:
https://github.com/project-oak/oak/blob/3935a27c710cd5ea98a382b69de282cd57b48e4f/oak_functions/examples/weather_lookup/module/src/tests.rs#L45-L50

Reply:
Done, thank you very much for this!